src/main/java/com/inturn/pfit/global/common/exception/define/ECommonErrorCode.java

```diff
@@ -0,0 +1,21 @@
+package com.inturn.pfit.global.common.exception.define;
+
+import com.inturn.pfit.global.common.dto.ErrorCodeDTO;
+import lombok.Getter;
+import org.springframework.http.HttpStatus;
+
+@Getter
+public enum ECommonErrorCode {
+
+	NOT_FOUND_EXCEPTION(ErrorCodeDTO.createErrorDTO(HttpStatus.NOT_FOUND.value(), "해당 데이터는 존재하지 않습니다.")),
```

Comment:
```suggestion
	NOT_FOUND_EXCEPTION(ErrorCodeDTO.createErrorDTO(HttpStatus.NOT_FOUND, "해당 데이터는 존재하지 않습니다.")),
```

enum을 사용한다는 전제하에 `.value()`를 제거할수 있을것 같아요 ㅎ

Reply:
네 반영하겠습니다!